src/planner/linux.rs

```diff
@@ -141,6 +131,29 @@ impl Planner for Linux {
             self.settings.ssl_cert_file.clone(),
         )?)
     }
+    async fn pre_uninstall_check(&self) -> Result<(), PlannerError> {
```

Comment:
Hmmmmm, why do we need to check for wsl1 / systemd before we uninstall?

Reply:
It's theoretically possible to change the wsl version of a distro!

Reply:
...I forgot about that (and really hate it lol).